### ISSUE TITLE
how to add TypiCms/Tags ? 

### ISSUE THREAD

**Issue by scil** [TypiCMS/Base#22]
after add provider,    run ``` php artisan```, following:
```
php artisan



  [Illuminate\Database\QueryException]
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'typ.typicms_tags
  ' doesn't exist (SQL: select `tag` from `typicms_tags`)






  [PDOException]
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'typ.typicms_tags
  ' doesn't exist



```

**Comment by sdebacker** [comment]
Did you try `php artisan vendor:publish`then `php artisan migrate`?

**Comment by sdebacker** [comment]
There is a known problem when installing Projects module, you have to manually copy migrations from Categories, Tags and Projects directory to the main migration directory, then enter `php artisan migrate` then add the Projects ModuleProvider class.

**Comment by sdebacker** [comment]
This is fixed in tags module version 2.4.2.
